app/policies/mixins/rbac_mixin.rb

```diff
@@ -4,12 +4,15 @@ module RBACMixin
 
     APPROVER_VISIBLE_STATES = [ApprovalStates::NOTIFIED_STATE, ApprovalStates::COMPLETED_STATE].freeze
 
-    def resource_check(verb, id = @record.id, klass = @record.class)
+    def resource_check(verb, record = @record)
       return true unless @user.rbac_enabled?
 
+      klass = record.class
+      return permission_check(verb, klass) unless [Request, Action].include?(klass)
```

Comment:
Don't need `permission_check` here, it's handled by `return false` when no proper role is found

Reply:
It is just a safe check, reminding this method is for Request and Action resource check.